src/ensembl/src/shared/components/instant-download/instant-download-gene/InstantDownloadGene.tsx

```diff
@@ -0,0 +1,206 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import intersection from 'lodash/intersection';
+import classNames from 'classnames';
+
+import { fetchForGene } from '../instant-download-fetch/fetchForGene';
+import { filterTranscriptOptions } from '../instant-download-transcript/InstantDownloadTranscript';
+
+import Checkbox from 'src/shared/components/checkbox/Checkbox';
+import InstantDownloadButton from '../instant-download-button/InstantDownloadButton';
+
+import styles from './InstantDownloadGene.scss';
+
+type Theme = 'light' | 'dark';
```

Comment:
Out of curiosity: does `InstantDownloadGene` even need a theme prop?

Reply:
I'll check with Andrea about this and see if we need this

Reply:
I asked Andrea about this. She said that we need the dark theme for the Genome browser zmenus:

```
Andrea Winterbottom  14:22
yes - but the download in the genome browser image should allow users to get the same data - however
at zoom where individual transcripts are being rendered, the download component in the zmenus should be the same as the transcript info panel in EV
and at zoom where the gene is being rendered as a block without the ability to identify individual transcripts, then only the gene download option should be available
```